This patch disables usage of simplecov, coveralls and fakeweb in the
test suite, and is intended to be removed once these packages enter
Debian.

Index: ruby-thor/spec/helper.rb
===================================================================
--- ruby-thor.orig/spec/helper.rb	2013-05-07 17:54:44.000000000 +0200
+++ ruby-thor/spec/helper.rb	2013-05-07 17:56:33.951392230 +0200
@@ -1,23 +1,15 @@
 $TESTING=true
 
-require 'simplecov'
-require 'coveralls'
-
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
-  SimpleCov::Formatter::HTMLFormatter,
-  Coveralls::SimpleCov::Formatter
-]
-SimpleCov.start
-
+puts File.join(File.dirname(__FILE__), "..", "lib")
 $:.unshift(File.join(File.dirname(__FILE__), "..", "lib"))
 require 'thor'
 require 'thor/group'
 require 'stringio'
 
-require 'rdoc'
+require 'rdoc/rdoc'
 require 'rspec'
 require 'diff/lcs' # You need diff/lcs installed to run specs (but not to run Thor).
-require 'fakeweb'  # You need fakeweb installed to run specs (but not to run Thor).
+#require 'fakeweb'  # You need fakeweb installed to run specs (but not to run Thor).
 
 # Set shell to basic
 $0 = "thor"
